app/1.0/configuration.md

```diff
@@ -1158,9 +1158,9 @@ The origins configuration can be useful in complex networking configurations, an
 
 The term origin (singular) refers to a particular scheme/host or IP address/port triple, as described in RFC 6454 (https://tools.ietf.org/html/rfc6454#section-3.2). In Kong's `origins` configuration, the scheme *must* be one of `http`, `https`, `tcp`, or `tls`. In each pair of origins, the scheme *must* be of similar type - thus `http` can pair with `https`, and `tcp` can pair with `tls`, but `http` and `https` cannot pair with `tcp` and `tls`.
 
-When an encrypted scheme like `tls` or `https` in the left origin is paired with an unencrypted scheme like `tcp` or `http` in the right origin, Kong will terminate TLS on incoming connections matching the left origin, and will then route traffic unencrypted to the specified right origin. This is useful when connections will be made to the Kong node over TLS, but the local service (for which Kong is proxying traffic) doesn't or can't terminate TLS. Similarly, if the left origin is `tcp` or `http` and the right origin is `tls` or `https`, Kong will accept unencrypted incoming traffic, and will then wrap that traffic in TLS as it is routed outbound. This capability is an important enabler of Kong Mesh.  
+When an encrypted scheme like `tls` or `https` in the left origin is paired with an unencrypted scheme like `tcp` or `http` in the right origin, Kong will terminate TLS on incoming connections matching the left origin, and will then route traffic unencrypted to the specified right origin. This is useful when connections will be made to the Kong node over TLS, but the local service (for which Kong is proxying traffic) doesn't or can't terminate TLS. Similarly, if the left origin is `tcp` or `http` and the right origin is `tls` or `https`, Kong will accept unencrypted incoming traffic, and will then wrap that traffic in TLS as it is routed outbound. This capability is an important enabler of Kong Mesh.
```

Comment:
What changed here?

Reply:
Nothing changed. It looks like a github fluke.